pmm-tests/minikube_operators_setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Prepare a set of base64 encoded values and non encoded for user and pass with administrator privileges to pmm-server (DBaaS)
+PMM_USER='admin';
+PMM_PASS='admin';
+
+PMM_USER_B64="$(echo -n "${PMM_USER}" | base64)";
+PMM_PASS_B64="$(echo -n "${PMM_PASS}" | base64)";
+
+# Install the PXC operator
+curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/pmm-branch/deploy/bundle.yaml | minikube kubectl -- apply -f -
+curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/pmm-branch/deploy/secrets.yaml | sed "s/pmmserver:.*=/pmmserver: ${PMM_PASS_B64}/g" | minikube kubectl -- apply -f -
+
+# Install the PSMDB operator
+curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v1.6.0/deploy/bundle.yaml | minikube kubectl -- apply -f -
```

Comment:
Should the version (v1.6.0) be saved to a variable for the purpose of re-using/changing it later?

Reply:
you have valid point, we should change it variable, I will make the required changes and push here.

Reply:
I have fixed this, and the pipeline now takes the operator version as a parameter